app/controllers/concerns/navigator_actions.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
+
+module NavigatorActions
+  extend ActiveSupport::Concern
```

Comment:
This shouldn't be a concern, as it's not a general purpose thing. We should make a base class in the navigator/ folder, and all navigator controllers should inherit from it. So put a base_controller.rb into the navigator folder, and then do 

```
module Navigator
  class ExperimentsController < BaseController
```